### PR TITLE
docs: 📝 Update team docs based on iteration retrospective 

### PR DESCRIPTION
**Pull request by lwjohnst86**
## Description

- This PR updates the  team docs based on the retrospective of the last iteration. 

## Reviewer Focus

This PR only needs a quick review.

### CODE REVIEW

**Review by philter87 — DISMISSED** [review]
Yay, Iterations and 1 approver 🎉

File: events/2023-06-in-person/index.qmd [hunk 1]
```diff
@@ -131,22 +131,22 @@ some topics and less on others.
             supplement with detailed enough comments, with a "top-level"
             comment describing the basic purpose, type of input, and
             type of output.
-    -   [ ] **Action**: Luke to put these into the contributing
+    -   **Action**: Luke to put these into the contributing
```
Comment (signekb) [inline events/2023-06-in-person/index.qmd:134]:
```suggestion
    -   **Action**: Luke will put these into the contributing
```

File: events/2023-10-in-person/index.qmd [hunk 2]
```diff
@@ -110,9 +110,9 @@ I'll be creating a blog post about this experience, so won't add much here expec
     - Could either hire them after we have a core functional box or hire them to help use develop our process and workflow (getting started, project management, teamwork practices, "how to work" rather than "what to work on")
     - If hired for process, we'll need to give them clear requirements for what we want from them
         - We might need some time to sit down and define the requirements for designing the how to work together
-    - [X] Action: To contact them about hiring them, what are the next steps? How do they charge for their services?
-    - [ ] Action: Need to spend some time writing doc on how to work together, the process. This could be the next “focus stage”, on starting to write out a design document
-    - [ ] Action: Make a detailed list of what outputs we would like from them
+    - Action: To contact them about hiring them, what are the next steps? How do they charge for their services?
```
Comment (signekb) [inline events/2023-10-in-person/index.qmd:113]:
```suggestion
    - **Action**: To contact them about hiring them, what are the next steps? How do they charge for their services?
```

**Review by signekb — COMMENTED** [review]
Nice 👍 Just have some minor comments and formatting suggestions.

File: events/2023-10-in-person/index.qmd [hunk 3]
```diff
@@ -177,21 +177,21 @@ I'll be creating a blog post about this experience, so won't add much here expec
     - Workflow, each project/package/app would have its own design doc and diagrams, etc?
 - Requirements for how to work together effectively as a team
     - Shorter-term roadmap (e.g. 3 month cycle, 6 month cycle, 1 year cycle)
-        - [X] Action: Luke update roadmap and convert into milestones on the docs but also on GitHub as a Milestone 
+        - Action: Luke update roadmap and convert into milestones on the docs but also on GitHub as a Milestone 
     - (see items above)
-    - [ ] Action: Setup pre-commit hooks?
+    - Action: Setup pre-commit hooks?
```
[the remark this answers is not in the document]
Comment (lwjohnst86) [reply events/2023-10-in-person/index.qmd:182]: @signekb we don't delete minutes, those are for archival purposes.

File: events/2023-10-in-person/index.qmd [hunk 4]
```diff
@@ -145,13 +145,13 @@ I'll be creating a blog post about this experience, so won't add much here expec
     - Building monthly sprints, with a period of roadmapping in those days
     - Use Kanban board with constrained time periods to use for the focus phases, with dedicated time pre-phase to describe and plan together what needs to be done and how long each task takes
     - Focus on smaller focused periods of working, with iterative development/feedback and smaller units of roadmapping.
-    - [ ] Action: Schedule time shortly 
+    - Action: Schedule time shortly
```
[the remark this answers is not in the document]
Comment (lwjohnst86) [reply events/2023-10-in-person/index.qmd:148]: I think it was related to scheduling the iteration, which we did 😛 Not a very good description for the minutes though!

File: events/2023-10-in-person/index.qmd (hunk 3 above)
[the remark this answers is not in the document]
Comment (signekb) [reply events/2023-10-in-person/index.qmd]: Ahhh, ofc not. I was a bit confused about whether this post was minutes or more general documentation

**Review by K-Beicher — COMMENTED** [review]
Looks fine to me

**Comment by lwjohnst86** [comment]
@K-Beicher if you approve it, can you click approve instead of comment? 🙏🏻

**Review by K-Beicher — APPROVED** [review]
I'll pick the right radio button this time :-)

**Pull request merged**